### PR TITLE
Logging and stuff

### PR DESCRIPTION
**Pull request by jotomo**

### CODE REVIEW

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/SerialIOThread.java [hunk 1]
```diff
@@ -158,22 +159,19 @@ public synchronized void sendMessage(MessageBase message) {
             mOutputStream.write(messageBytes);
         } catch (Exception e) {
             log.error("sendMessage write exception: ", e);
-            e.printStackTrace();
+            log.error("Unhandled exception", e);
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/SerialIOThread.java]: probably unnecessary

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/SerialIOThread.java [hunk 2]
```diff
@@ -158,22 +159,19 @@ public synchronized void sendMessage(MessageBase message) {
...
         }
 
         synchronized (message) {
             try {
                 message.wait(5000);
             } catch (InterruptedException e) {
                 log.error("sendMessage InterruptedException", e);
-                e.printStackTrace();
+                log.error("Unhandled exception", e);
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaR/SerialIOThread.java]: probably unnecessary

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRKorean/SerialIOThread.java [hunk 3]
```diff
@@ -159,22 +160,19 @@ public synchronized void sendMessage(MessageBase message) {
             mOutputStream.write(messageBytes);
         } catch (Exception e) {
             log.error("sendMessage write exception: ", e);
-            e.printStackTrace();
+            log.error("Unhandled exception", e);
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRKorean/SerialIOThread.java]: probably unnecessary

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRKorean/SerialIOThread.java [hunk 4]
```diff
@@ -159,22 +160,19 @@ public synchronized void sendMessage(MessageBase message) {
...
         }
 
         synchronized (message) {
             try {
                 message.wait(5000);
             } catch (InterruptedException e) {
                 log.error("sendMessage InterruptedException", e);
-                e.printStackTrace();
+                log.error("Unhandled exception", e);
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRKorean/SerialIOThread.java]: probably unnecessary

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRv2/SerialIOThread.java [hunk 5]
```diff
@@ -159,22 +160,19 @@ public synchronized void sendMessage(MessageBase message) {
             mOutputStream.write(messageBytes);
         } catch (Exception e) {
             log.error("sendMessage write exception: ", e);
-            e.printStackTrace();
+            log.error("Unhandled exception", e);
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRv2/SerialIOThread.java]: probably unnecessary

File: app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRv2/SerialIOThread.java [hunk 6]
```diff
@@ -159,22 +160,19 @@ public synchronized void sendMessage(MessageBase message) {
...
         }
 
         synchronized (message) {
             try {
                 message.wait(5000);
             } catch (InterruptedException e) {
                 log.error("sendMessage InterruptedException", e);
-                e.printStackTrace();
+                log.error("Unhandled exception", e);
```
Comment (MilosKozak) [inline app/src/main/java/info/nightscout/androidaps/plugins/PumpDanaRv2/SerialIOThread.java]: probably unnecessary

**Comment by jotomo** [comment]
Thanks - fixed now.

**Pull request merged**